packages/components/src/components/Label/Label.module.scss

```diff
@@ -10,4 +10,13 @@
   &.disabled {
     color: var(--label--color--disabled);
   }
+
+  > div {
+    display: inline-flex;
+    order: 1;
+  }
+
+  & .optional {
+    order: 0;
+  }
```

Comment:
The `> div` selector is unspecific and may match accidentally other elements.

You could introduce a new tunnel exit for `contentRight` and put it in it's own flex-aligned div with class name e.g. `.contentRight`.

If you also want users to put stuff into the tunnel, you can use a `slot` prop and the existing props context, to assign the `tunnelId`.